ksqldb-engine/src/main/java/io/confluent/ksql/engine/KsqlEngine.java

```diff
@@ -324,6 +324,15 @@ public TransientQueryMetadata executeTransientQuery(
     }
   }
 
+  public KsqlConfig getKsqlConfig() {
+    return this.ksqlConfig.cloneWithoutOverride();
```

Comment:
why do you have to clone it? `KsqlConfig` is immutable.

Reply:
My build was failing when I exposed `KsqlConfig` without cloning it as "internal implementation is exposed". Is there another way to do this?